snapd-glib/snapd-snap.c

```diff
@@ -8,6 +8,8 @@
  */
 
 #include "snapd-snap.h"
+#include "glib-object.h"
+#include "glib.h"
```

Comment:
This shouldn't be required.

Reply:
very sorry all these got auto added by vscode and I missed it.